werkit/aws_lambda/test_integration.py

```diff
@@ -0,0 +1,156 @@
+import os
+import base64
+import boto3
+import uuid
+import zipfile
+import tempfile
+import json
+from pprint import pprint
+
+role = "arn:aws:iam::139234625917:role/werkit-test-integration"
+# This role has the following policy: AWSLambdaRole
+
+path_to_orchestrator_zip = "/tmp/python-orchestrator.zip"
+path_to_worker_zip = "/tmp/python-worker.zip"
+
+
+# https://stackoverflow.com/a/1855118/366856
+def zipdir(path, ziph):
+    # ziph is zipfile handle
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            ziph.write(os.path.join(root, file))
```

Comment:
I think if you use [the `arcname` arg to `ziph.write()`](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.write) you won't need to chdir before you call this.

e.g.
```py
             ziph.write(os.path.join(root, file), arcname=file)
```

Reply:
Fixed in 5cbf2f38d0cebf4240abbc12753f29b27fc8e014